sksurv/tree/tree.py

```diff
@@ -229,6 +236,11 @@ def fit(self, X, y, sample_weight=None, check_input=True):
         # one "class" for CHF, one for survival function
         self.n_classes_ = np.ones(self.n_outputs_, dtype=np.intp) * 2
 
+        if self.low_memory:
```

Comment:
an alternative here would be
```
if self.low_memory:
    ...
else:
    ...
```
and likewise elsewhere

Reply:
I would prefer if-else